steps/aws-secrets-manager/1.0.0/step.yml

```diff
@@ -0,0 +1,62 @@
+title: Fetch secrets
```

Comment:
Please include the underlying service behind this step, something like: `Fetch secrets from AWS Secrets Manager`.

https://devcenter.bitrise.io/contributors/create-your-own-step/#the-title:
`... Include the name of the service and the function it fulfills, such as Git Clone. ...`

Reply:
Addressed in moneylion/bitrise-step-aws-secrets-manager@b843ca6bebd22d36aef62bd358ee0ece2dd01901.